src/tests/access/test_dual_accesscontrol.cairo

```diff
@@ -63,7 +63,7 @@ fn test_dual_supports_interface() {
 }
 
 #[test]
-#[ignore] // REASON: inconsistent ENTRYPOINT_NOT_FOUND panic message
+#[ignore] // REASON: foundry entrypoint_not_found error message inconsistent with mainnet.
```

Comment:
```suggestion
#[ignore] // REASON: should_panic attribute not fit for complex panic messages.
```

This test can't be fixed even if the mainnet error was the same as the foundry one, because the panic message is thrown in the second call, and the dual dispatchers unwrap the syscall result. We would need to update the dual dispatchers to return the error to catch it, or use the should panic attribute to match the error.

Reply:
The same applies for the other tests expecting ENTRYPOINT_NOT_FOUND

Reply:
Fixed